pegasus/sites.v3/code.org/public/curriculum/computer-vision.haml

```diff
@@ -1,75 +1,137 @@
-    .action-block.action-block--two-col
-      %img{src: "/shared/images/social-media/ai-101-social.png", alt: "", class: "col-50"}
-      .text-wrapper.col-50
-        %h3.heading-sm
-          =hoc_s(:csa_ai_page_learn_more_title)
-        %p=hoc_s(:csa_ai_page_learn_more_description)
-        %a.link-button{href: "/ai/pl/101"}
-          =hoc_s(:csa_ai_page_learn_more_link)
-
-= view :section_divider_line
-
-%section.no-padding-bottom.faq__section
-  .wrapper
-    %h2=hoc_s(:heading_faq)
-    = view :faq_computer_vision
-
-%section
-  .wrapper
-    = view :support_and_questions_section
 
+-# TODO Remove this DCDO flag logic after the 2024 curriculum launch
+- if !!DCDO.get('curriculum-launch-2024', false)
+  %link{href: "/css/generated/page/computer-vision-styles.css", rel: "stylesheet"}
+
+  %section.hero-banner-basic.overlay
+    .wrapper.grid-container.justify-space-between.align-items-center.gap-4.wider-text
+      .text-wrapper
+        %h1.white
+          =hoc_s("computer_vision_page.heading")
+        %p.body-two.white
+          =hoc_s("computer_vision_page.top.desc")
+        %a.link-button.white{href: "#curricula"}
+          =hoc_s("computer_vision_page.top.button")
+      %figure
+        %img{src: "/images/banners/ai-bot-vision.svg", alt: "", style: "width: 100%"}
+
+  %section
+    .wrapper.centered
+      %h2=hoc_s("computer_vision_page.why.heading")
+      %p=hoc_s("computer_vision_page.why.desc")
+      = view :"computer_vision/computer_vision_why_list"
+      .button-wrapper.centered
+        %a.link-button{href: "#curricula"}
+          =hoc_s("computer_vision_page.why.button.explore_curriculum")
+        %a.link-button.secondary{href: "#professional-learning"}
+          =hoc_s("computer_vision_page.why.button.see_pl")
+
+  %section.bg-primary-light
+    .wrapper
+      .flex-container.align-items-center.flex-direction-column-mobile.flex-wrap.gap-2
+        .text-wrapper.flex-2
+          %h2.heading-sm
+            =hoc_s("game_design_page.resources.heading")
+          %p.no-margin-bottom
+            =hoc_s("game_design_page.resources.desc", markdown: :inline)
+        %a.link-button{href: CDO.studio_url("/users/sign_up")}
+          =hoc_s("game_design_page.resources.button")
+
+  %section#curricula
+    .wrapper
+      %h2=hoc_s("computer_vision_page.curricula.heading")
+      = view :"computer_vision/computer_vision_curricula"
+      %hr.light
+      #professional-learning
+        %h2=hoc_s("computer_vision_page.teach.heading")
+        %p=hoc_s("computer_vision_page.teach.desc")
+        = view :"computer_vision/computer_vision_teach"
+
+  %section.bg-neutral-light
+    .wrapper.centered
+      %h2=hoc_s("computer_vision_page.learn_more.heading")
+      = view :"computer_vision/computer_vision_learn_more"
+      %hr.light
+      %h3=hoc_s("computer_vision_page.learn_more.discover.heading")
+      %p=hoc_s("computer_vision_page.learn_more.discover.desc")
+      %a.link-button.secondary{href: "/ai"}
```

Comment:
Nit: Should this have `.white` (or some other color) to contrast the gray background?

Reply:
Nope! The secondary buttons have a transparent bg, but there is a `.white` version that turns the outline and text white if it's on a dark background.

Reply:
Oh gotcha!